rs/execution_environment/src/scheduler.rs

```diff
@@ -1967,6 +1969,7 @@ fn observe_replicated_state_metrics(
     let mut canisters_with_old_open_call_contexts = 0;
     let mut old_call_contexts_count = 0;
     let mut num_stop_canister_calls_without_call_id = 0;
+    let mut open_signature_request_contexts_by_key_id = BTreeMap::<MasterPublicKeyId, u32>::new();
```

Comment:
```suggestion
    let mut in_flight_signature_request_contexts_by_key_id = BTreeMap::<MasterPublicKeyId, u32>::new();
```
"in flight" sounds a bit better imo, if you agree then I would apply similar changes to metric description and so on.

Reply:
Done 👍